airbyte-ci/connectors/pipelines/pipelines/dagger/containers/java.py

```diff
@@ -172,4 +176,10 @@ async def with_airbyte_java_connector(context: ConnectorContext, connector_java_
         .with_label("io.airbyte.name", context.metadata["dockerRepository"])
         .with_entrypoint(entrypoint)
     )
-    return await finalize_build(context, connector_container)
+    connector_container = await finalize_build(context, connector_container)
+    connector_container = (
+        connector_container.with_exec(["chown", "-R", user, "/airbyte"], skip_entrypoint=True)
+        .with_exec(["chmod", "777", "/tmp"], skip_entrypoint=True)
+        .with_user(user)
+    )
```

Comment:
I'm not entirely sure why this is needed (shouldn't every user have r/w access to /tmp?), but it is.

Reply:
I thought too, but I faced permission denied errors while writing to /tmp during the integration test run on S3